### DISCUSSION TITLE
Different Chocolatey sources depending on task (Discover vs Update)

### DISCUSSION THREAD

**Discussion by Infinitay** [Devolutions/UniGetUI#3034] [Q&A] [answered]
Before I continue, here is my current Package Manager settings for Chocolatey:

![image](https://github.com/user-attachments/assets/eb1a4d59-724b-4a2c-86df-2af86046f502)

I'm not sure if this is particularly an issue or not so I'm opening up a discussion post for more information. I noticed just now that depending on what action I'm trying to do within WinGetUI, the chocolatey source changes. For example, when I go to **Software Updates** I only see `Chocolatey: community` packages. I should note that I haven't used WinGetUI to install any new chocolatey packages as I rely either on chocolatey CLI or chocolatey GUI for that. However, whenever I go to **Discover Packages** it lists packages only with the `Chocolatey: chocolatey` source. 

For me, this is causing a minor issue to because the packages listed from `Chocolatey: chocolatey` seem to be outdated compared to packages found within `Chocolatey: community` - despite having the same package id/maintainer(?).

# Discover Packages

![image](https://github.com/user-attachments/assets/badecad1-1d87-4049-a73d-a2fabba18e86)

# Software Updates

![image](https://github.com/user-attachments/assets/d763e4f9-9d06-438e-8d5d-97311c03ca82)

**Comment by marticliment** [comment, marked as answer]
I have opened a bug ticket for your issue: https://github.com/marticliment/UniGetUI/issues/3036

**Reply by Infinitay** [reply]
Thank you for your help. I didn't realize it was in fact a bug.
